cmd/controller/main.go

```diff
@@ -25,47 +25,64 @@ import (
 var version = "dev"
 var name = "deviceplane-controller"
 
+//commit for test
 var (
 	addr = kingpin.
 		Flag("addr", "").
 		Default(":8080").
+		OverrideDefaultFromEnvar("ADDR").
 		String()
 	mysql = kingpin.
 		Flag("mysql", "").
 		Default("deviceplane:deviceplane@tcp(localhost:3306)/deviceplane?parseTime=true").
+		OverrideDefaultFromEnvar("MYSQL_CONNECTION_STRING").
```

Comment:
Should we keep this in line with the other variables and name it `MYSQL` w/out the connection string bit?